proc/foxsi_level0_to_level1.pro

```diff
@@ -126,6 +142,25 @@ FUNCTION	FOXSI_LEVEL0_TO_LEVEL1, FILENAME, DETECTOR = DETECTOR, STOP = STOP, $
 
 	; Identifying hits, associated values, and unrelated values...
 
+  ; identify in different cases in nside (ASIC 0 and 1) are on y axis (value=1) or x axis (value=0)
+  ; and what are the shift to associate XY detector coordinates
+  IF CDTE EQ 0 THEN BEGIN
+	  nside = 1
+	  pside = 0
+	  strip_shift = [127,63]
+	ENDIF ELSE BEGIN
+	 ; IF year EQ 2014 THEN BEGIN
+	 ;   nside = 0
+	 ;   pside = 1
+	 ;   strip_shift = [63,127]
+	 ; ENDIF 
+	 ; IF year EQ 2018 THEN BEGIN
+	    nside = 1
+	    pside = 0
+	    strip_shift = [127,63]
+	 ; ENDIF
+	ENDELSE
```

Comment:
It looks like the result is the same whether CdTe is set to 0 or 1 .

Reply:
that is very true. I will delete these lines.